wandb/sdk/launch/agent/README.md

```diff
@@ -0,0 +1,3 @@
+An agent is broadly defined as a *"program that collects information or performs a task in the background at a particular schedule. The term agent is often thought of as a software abstraction that is capable of acting with a certain degree of autonomy to perform a particular task on behalf of its host"* ([Source](https://www.techopedia.com/definition/1292/agent#:~:text=Techopedia%20Explains%20Agent-,What%20Does%20Agent%20Mean%3F,on%20behalf%20of%20its%20host.))
```

Comment:
do we need this predicate stuff? Kinda fun but are techies really going to spend the time reading? Might be annoying idk

Reply:
Under this umbrella definition the launch agent, sweeps agent & an RL agent, all work 😃